File: src/person/Student.js
var Knowledge = require('./consciousness/Knowledge');

class Student {
    constructor(studentName, level) {
        this.studentName = studentName;
        this.studentKnowledge = new Knowledge(level)
    }

    setKnowledge(knowledge) {
        //Sets knowledge???
    }

    getKnowledge(){
        return this.studentKnowledge.getLevel();
    }
    getName(){
        return this.studentName;
    }
}
module.exports = Student;

File: src/person/Student.java
package person;

import person.consciousness.Knowledge;

public class Student {
    private String studentName;
    private Knowledge studentKnowledge;

    public Student(String studentName, int level) {
        //Constructor
        this.studentName = studentName;
        this.studentKnowledge = new Knowledge(level);
    }

    public void setKnowledge(Knowledge knowledge) {
        //Sets knowledge???
    }

    public int getKnowledge(){
        return this.studentKnowledge.getLevel();
    }
    
    public String getStudentName() {
    	return this.studentName;
    }
}
